### PR TITLE
Remove PrivacyTools

### PR DESCRIPTION
**Pull request by HyperLemons**
https://www.privacyguides.org/en/about/privacytools/

if its too long to read just go to their vpn page and you can see they are a clear nordvpn shill lol
image

### CODE REVIEW

**Comment by hopolapopola** [comment]
I'm concerned that someone might've hacked into Daniel's GitHub to leave a comment that has zero evidence nor evaluates a technical problem from a technical angle, as I have seen the real Daniel do on many occasions. 

That aside, what the person who has gotten into Daniel's account is saying to defend a paid sponsor-riddled borderline scam website is mostly provably false. Anyone could go to the PrivacyGuides recommendations and see that the only time Apple products (and notably, *not* services) are recommended are when people are using Apple hardware and OSes to begin with. No Apple OS is recommended by PrivacyGuides, nor are iPhones or Macs recommended on the PrivacyGuides website. The claims of harassment are something that the real Daniel also makes but I am not about to victim-blame him nor dispute those, as he and the accused are the only ones who truly know the veracity of those claims.

**Comment by eylenburg** [comment]
I think Privacytools has some value for example they have more recommendations for other categories (e.g. Maps apps). But I had another look at their website and the first thing is "Top 5 VPNs" (of which 3 are the usual sponsored subjects). I think that despite its flaws, PrivacyGuides is the superior guide, with no sponsored entries and clear criteria for each category. I also know about [PRISM-break](https://prism-break.org/en/all/) which seems fine. But both Privacytools and PRISM-break are a bit outdated, e.g. Privacytools still recommends LocalMonero (which shut down recently) and PRISM-break still recommends Privacy Badger.

**Comment by hopolapopola** [comment]
> Worth noting @hopolapopola is banned from the GrapheneOS community for previously engaging in harassment.

Worth noting *you* sent me threatening messages implying legal action purely because I help moderate PG's matrix rooms despite never saying anything bad about you. If you have actual evidence of harassment by me, please send it to my email ghost@arachnid.pink and I will go back and delete any messages you believe were harassment.

**Comment by hopolapopola** [comment]
Please refrain from continuing these libellous claims against me or I will actually retain a lawyer, My threat is not as empty as yours, I have the money for it. So please, either provide evidence that I personally have said anything that could be construed as harassment or desist with this bullshit.

**Comment by hopolapopola** [comment]
Show me the evidence of where I have said that I, @hopolapopola, support anything that you allege Jonah has done. Show me even a single message.

**Comment by hopolapopola** [comment]
I have already provided an email where you can send any evidence.

**Comment by hopolapopola** [comment]
I'd have to check but there is at least one instance where I deleted linking to KF. So again, show me evidence where *I* have supported harassment.

**Comment by hopolapopola** [comment]
Okay so no, you don't have evidence of me actually engaging in what could be defined as harassment. I really don't want to go through courts over this, but you're really pushing my buttons.

**Comment by hopolapopola** [comment]
You cannot possibly hold me responsible for every single message in an online chat, as if I read every single message upon checking Matrix. That is patently ridiculous.

**Comment by hopolapopola** [comment]
Jonah and the other mods are NOT my friends. You are doing guilt by association, sending threatening direct messages to people, and being terminally online. You can have the last word, but I **will** take legal action if you libel me again elsewhere, so please, don't. I would much rather you spend your money on developing Graphene than lawyers.

**Comment by dngray** [comment]
The screenshot you provided is not evidence of anything. Yes some people a long time noticed the thread about you. Everyone who knows what you're like knows about it (including the grapheneos moderators no doubt). You wouldn't say they're participating in it just because they know about it.

I do think it's laughable that you think I'm participating simply by saying their site was slow. If I remember correctly I didn't even wait for it to finish loading because I got bored.

**Comment by dngray** [comment]
The screenshot you provided is not evidence of anything. Yes some people a long time ago noticed the thread about you. Everyone who knows what you're like knows about it (including the GrapheneOS moderators no doubt). You wouldn't say they're participating in it just because they know about it. In fact it's so old news that nobody even talks about it anymore, except for when you mention it.

I do think it's laughable that you think I'm participating simply by saying their site was slow. If I remember correctly I didn't even wait for it to finish loading because I got bored.

As for the comments about "ban evading" that simply was that if you use GrapheneOS and want support for it, use a unique name not used in the PG rooms, because you ban people simply for being in both communities. Unless people totally agree 100% with you, you ban them. If they bring any attention to your unsubstantiated claims you ban them. Nothing said on our forums is libel or harassment against you. We've been supportive of the GrapheneOS project and continue to recommend it.

TLDR: unsubstantiated moaning about Jonah and Techlore. Which by the way, is about the only thing I mentioned on the forums when you made me the ultimatum that I had to "fire" Jonah from his own project Privacy Guides or be banned from GrapheneOS community. As there has never been any evidence of your claims or violations of our [CoC](https://www.privacyguides.org/coc) there isn't anything to action.

**Comment by eylenburg** [comment]
I'm not associated with any of the sites mentioned and dont want to be pulled into any drama. I'd appreciate of you all could delete any off topic comments or I will do myself later, sorry
